applications/dashboard/controllers/api/AuthenticateApiController.php

```diff
@@ -459,4 +459,44 @@ public function ssoDataSchema() {
 
         return $ssoDataSchema;
     }
+
+    /**
+     * Authenticate a user with username/email and password.
+     *
+     * @param array $body The username/password to sign in.
+     * @return array Returns the signed in user.
+     * @throws \Exception Throws all exceptions to be dispatched as error responses.
+     */
+    public function post_password(array $body): array {
+        $this->permission();
+
+        $in = $this->schema([
+            'username:s' => 'The user\'s Username or email address.',
+            'password:s' => 'The user\'s password.',
+            'persist:b' => [
```

Comment:
Shouldn't we also add the setCookie option like we did for `post()`?

Reply:
I had thought we removed that. It's not in the markdown anymore at least.